persist-cli/src/main/java/io/ballerina/persist/cmd/Generate.java

```diff
@@ -154,6 +156,17 @@ public void execute() {
             } catch (BalException e) {
                 errStream.printf("ERROR: failed to generate types and client for the definition file(%s). %s%n",
                         file.getFileName(), e.getMessage());
+                return;
+            }
+
+            try {
+                ArrayList<Entity> entityArray = new ArrayList<>(entityModule.getEntityMap().values());
+                String[] sqlScripts = SqlScriptGenerationUtils.generateSqlScript(entityArray);
+                SqlScriptGenerationUtils.writeScriptFile(entityModule.getModuleName(), sqlScripts,
+                        generatedSourceDirPath);
+            } catch (BalException e) {
+                errStream.printf("ERROR: failed to generate SQL schema for the definition file(%s). "
+                        + e.getMessage() + "%n", file.getFileName());
```

Comment:
Why are we apeending string when we can use printf as used in the start?

Reply:
updated in [345097c](https://github.com/ballerina-platform/persist-tools/pull/140/commits/345097c9dba308883a3b916944e36e8ab3f03e74)